flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/Fabric8FlinkKubeClientTest.java

```diff
@@ -630,4 +630,37 @@ private KubernetesConfigMap buildTestingConfigMap() {
                         .withData(data)
                         .build());
     }
+
+    @Test
+    void testMockPrePreparedResources() throws Exception {
+        // regenerate a test JM spec
+        Deployment testDeployment = kubernetesJobManagerSpecification.getDeployment();
+        List<HasMetadata> testAccompanyingResources =
+                kubernetesJobManagerSpecification.getAccompanyingResources();
+        List<HasMetadata> mockPrePreparedResources = new ArrayList<>();
+        for (int i = 0; i < 3; i++) {
+            Pod mockPod =
+                    new PodBuilder()
+                            .editOrNewMetadata()
+                            .withName(String.format("mock-pod-preprepared-resource%d", i))
```

Comment:
```suggestion
                            .withName("mock-pod-preprepared-resource-" + i)
```

Reply:
hmmm, does this kind of string is benefit in Java coding?

Reply:
1) It is easier to read
2) it is faster (`String.format()` is slower!), but this is not really important for this test

Reply:
OK. I will update it. ;-)